RLBotPack/Levi/requirements.txt

```diff
@@ -0,0 +1,4 @@
+numpy
+RLUtilities
+torch==1.6.0+cpu
+-f https://download.pytorch.org/whl/torch_stable.html
```

Comment:
Thanks for including the requirements.txt! Have you confirmed whether torch installs cleanly from RLBotGUI when a user opts in?

Reply:
i will do another check

Reply:
seems to work perfectly